app/models/shortener/shortened_url.rb

```diff
@@ -46,7 +46,9 @@ def self.generate(orig_url, user=nil, project_id)
 
   # the create method changed in rails 4...
   CREATE_METHOD_NAME =
-    if Rails::VERSION::MAJOR >= 4
+    if Rails::VERSION::MAJOR >= 5
```

Comment:
Since this is our fork of shortner and it will always match our current Rails version...could we just simplify all of this?

```ruby
CREATE_METHOD_NAME = "_create_record"
```

Reply:
Yea, if it's just our library, then we don't need any of the logic. None of it applies to Rails 5 and up.

I'll push a new version and make sure the tests on the main app don't regress.